rerun_notebook/callbacks.ipynb

```diff
@@ -67,24 +73,46 @@
     "        self.selection_label.value = f\"selected=[{self.selection}]\"\n",
     "\n",
     "    def on_selection_change(self, selection) -> None:\n",
+    "        # Because it is possible to select multiple items at a time in the Viewer,\n",
+    "        # the selection change event yields a list of items.\n",
     "        label = []\n",
     "        for item in selection:\n",
     "            if item.kind == \"entity\":\n",
-    "                label.append(f\"Entity({item.entity_path})\")\n",
-    "            elif item.kind == \"instance\":\n",
-    "                label.append(f\"Entity({item.entity_path}[{item.instance_id}])\")\n",
+    "                # Entities always include their entity path,\n",
+    "                # and optionally also:\n",
+    "                # * The instance ID\n",
+    "                # * The mouse coordinates in _scene space_ at the time of the selection\n",
+    "                #   in a 2D or 3D view. Scene space here means in the coordinate system\n",
+    "                #   of the view and relative to its origin.\n",
```

Comment:
Personally, I talk a lot about "scene space" but I don't think we defined that anywhere aaand it sounds an awful lot like "screen space", so how about:
```suggestion
    "                # * The coordinates at which a selection occurred for 2D or 3D views. \n",
    "                #    Coordinates are within the view's space and thus relative to its origin.\n",
```
(also dropping the "mouse" part to avoid confusion with mouse coordinates)